### ISSUE TITLE
Tutorial on colab, dont understand: oi = get_implementation_from_shorthand("fbbt.obo")

### ISSUE THREAD

**Issue by raven44099** [INCATools/ontology-access-kit#467]
I followed the tutorial [[Part 2: Basic Python](https://incatools.github.io/ontology-access-kit/intro/tutorial02.html)  ], except I install poetry differently. Moreover, I'm not sure where to put the file `fbbt.obo`. Does "top level directory" mean inside the "my-oak-demo" project?

When I put it there, I get an error, since the function `get_implementation_from_shorthand()` is not defined, which makes sense. How do I correct this error?

Here is the code which  I used:
```py
# !curl -sSL https://install.python-poetry.org | python3 -
# !curl -sSL https://install.python-poetry.org
!pip install -q --pre poetry
!poetry --version

!poetry new --src -n my-oak-demo
%cd my-oak-demo
!poetry add oaklib

!wget http://purl.obolibrary.org/obo/fbbt.obo

oi = get_implementation_from_shorthand("fbbt.obo")
```
__Gives Error:__
```py
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
[<ipython-input-33-8061706781b9>](https://localhost:8080/#) in <module>
----> 1 oi = get_implementation_from_shorthand("fbbt.obo")

NameError: name 'get_implementation_from_shorthand' is not defined
```

**Comment by cmungall** [comment]
Hi @raven44099!

You need to first import the method

**Comment by raven44099** [comment]
I see. 
The tutorial code at the above link has been updated to `oi = get_adapter("fbbt.obo")`, however, the line `from oaklib import get_adapter` is still missing. When I added that line, it worked!

Interestingly,  `from oaklib import get_adapter` is stated in the example [_ontology-access-kit/blob/main/README.md_ ](https://github.com/INCATools/ontology-access-kit/blob/main/README.md), but it should be added to the tutorial as well, as one can not rely on users checking the README.md as the first  place of search, although it suggests itself.
